packages/webcert/src/feature/certificate/Inputs/UeCauseOfDeath/UeCauseOfDeathList.tsx

```diff
@@ -133,7 +133,11 @@ const UeCauseOfDeathList: React.FC<Props> = ({ question, disabled }) => {
                   validationErrors={validationErrors.filter((v) => v.field.includes(`[${index}]`))}>
                   <DeleteButtonWrapper className="iu-ml-500">
                     {index > 0 && (
-                      <CustomButton disabled={disabled} buttonStyle="secondary" onClick={() => handleDeleteRow(config.id)} height="47px">
+                      <CustomButton
+                        disabled={disabled}
+                        buttonStyle="secondary"
+                        onClick={() => handleDeleteRow(config.id)}
+                        height="2.956rem">
```

Comment:
I think this wil probably not be the only case when we will need an in-line button.
I would prefer not to set the height here, but rather maybe a separate control for inline button.